### PR TITLE
Added tag prefix support for aws/cloud costs

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: plugins/aws-costs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.3.1
```
Comment (rbren) [inline plugins/aws-costs/CHANGELOG.md:3]: let's make this 1.4

File: plugins/aws-costs/aws-costs.sh [hunk 2]
```diff
@@ -81,6 +83,10 @@ fi
 initial_date_time=$(date -u -d  $days+' day ago' +"%Y-%m-%d %H:00:00.000")
 final_date_time=$(date -u +"%Y-%m-%d %H:00:00.000")
 
+if [[ "$tagprefix" = "" ]]; then
+  tagprefix = "resource_tags_user_"
+fi
```
Comment (rbren) [inline plugins/aws-costs/aws-costs.sh:88]: instead of doing this here, let's do it in the helm chart. That way the user can still unset it.

**Review by rbren — APPROVED** [review]

File: plugins/aws-costs/CHANGELOG.md (hunk 1 above)
Comment (jdesouza) [reply plugins/aws-costs/CHANGELOG.md]: done

File: plugins/aws-costs/aws-costs.sh (hunk 2 above)
Comment (jdesouza) [reply plugins/aws-costs/aws-costs.sh]: done

**Pull request merged**